tests/unit/test-util/test.adb.js

```diff
@@ -780,8 +781,8 @@ describe('utils/adb', () => {
 
       const promise = adbUtils.startFirefoxAPK(
         'device1',
-        'org.mozilla.geckoview_example',
-        'org.mozilla.geckoview_example.GeckoViewActivity', // firefoxApkComponent
+        'org.mozilla.geckoview_example.release',
+        'GeckoViewActivity', // firefoxApkComponent
```

Comment:
Those tests should not be modified (because the old syntax should still work).

Please add a new test.

Reply:
Sure thing - I'll add a new test for `org.mozilla.fenix.nightly`